oauth2_http/java/com/google/auth/oauth2/MetricsUtils.java

```diff
@@ -67,4 +71,50 @@ private static String getAuthLibraryVersion() {
     }
     return version;
   }
+
+  public enum RequestType {
+    ACCESS_TOKEN_REQUEST("at"),
+    ID_TOKEN_REQUEST("it"),
+    METADATA_SERVER_PIN("mds"),
+    UNSPECIFIED("unspecified"),
+    UNTRACKED("untracked");
```

Comment:
Can you add a comment about the difference between UNSPECIFIED and UNTRACKED for future development? 

If I'm understanding it correctly, `UNSPECIFIED` is only omit the  `auth-request-type/REQUEST_TYPE` portion, but `UNTRACKED` omits the entire header?